lib/OpusVL/FB11/Auth/Brain.pm

```diff
@@ -22,18 +55,127 @@ has short_name => (
     default => 'fb11-auth',
 );
 
+=head1 HATS AND SERVICES
+
+=head2 auth
+
+This Brain wears an auth hat and provides the auth service. See
+L<OpusVL::FB11::Auth::Brain::Hat::auth>.
+
+This is essentially a stand-in for a future auth service that has not been
+designed yet.
+
+=head2 fb11authdb
+
+This is an alias into the auth hat and service.
+
+=head2 dbicdh::manager
+
+The best way to ensure your schemata get deployed is to have this brain in your
+hive, since it provides the C<dbicdh::manager> service.
+
+Remember that nothing actually accesses this service by default: this is
+because, in principle, FB11 should not require a database to run.
+
+=head2 dbicdh::consumer
+
+The FB11AuthDB is deployed and upgraded as the zeroth priority schema, and thus
+is both the manager and consumer.
+
+=cut
+
 # TODO retest legacy parameters against someone who has them
 sub hats {
     (
         qw/auth parameters/,
         fb11authdb => {
             class => 'auth'
-        }
+        },
+        'dbicdh::consumer' => {
+            class => '+OpusVL::FB11::Hat::dbicdh::consumer::is_brain',
```

Comment:
I think you're asking about the core FB11 brain, which doesn't exist. if we were to make one I'd probably recommend calling it OpusVL::FB11::Brain, a convention we've adopted in many other places, meaning the namespace there would be OpusVL::FB11::Brain::Hat::*

There is still scope for confusion, though; maybe we need a namespace that communicates this is a stash of useful hats instead of a place full of hats with assigned purpose.

Reply:
We'll fix this in #20